timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -372,9 +378,6 @@ def _get_model_params(self) -> dict:
                 callbacks=self.callbacks,
             )
         )
```

Comment:
I think we can fold `freq`, `prediction_length` and `quantiles` into `_get_default_params` as well.

As for callbacks, we can probably just directly set them in `_get_estimator` as
```python
        default_trainer_kwargs = {
           ...
            "callbacks": self.callbacks,
           ...
        }
```